project-anemone/app/Http/Controllers/Api/V1/Admin/Api.yaml

```diff
@@ -338,19 +191,6 @@ paths:
         "400":
           description: "Invalid username/password supplied"  
   /logout:
-    get:
-      tags:
-      - "user"
-      summary: "Logs out current user session"
-      description: ""
-      operationId: "logout"
-      produces:
-      - "application/xml"
-      - "application/json"
-      parameters: []
```

Comment:
This doesn't look right to me, perhaps this is a copy/pasta? 
line 207 (`parameters: []`) under `/logout` can be removed